### PR TITLE
Fix package ordering

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala [hunk 1]
```diff
@@ -118,7 +118,13 @@ final class DefaultUniverseClient(
           }
         } map { case (contentType, bodyInputStream) =>
           try {
-            decodeUniverse(contentType, bodyInputStream, repository.uri)
+            universe.v3.model.Repository(
+              decodeUniverse(
+                contentType,
+                bodyInputStream,
+                repository.uri
+              ).packages.sorted.reverse
```
Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: The `decodeUniverse` method already returns a `universe.v3.model.Repository`, but you're unwrapping the packages, sorting them, and then wrapping them in `Repository` again. Should the sorting be moved into the `decodeUniverse` method instead?

**Comment by jsancio** [comment]
Done.

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala (hunk 1 above)
Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: @cruhland Done. Thanks for the review!

**Comment by cruhland** [comment]
LGTM when tests pass!

**Pull request merged**